ilc/package.json

```diff
@@ -4,15 +4,16 @@
   "description": "",
   "main": "server/index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test mocha",
+    "test": "npm run build:server && cross-env NODE_ENV=test mocha",
     "test:ci": "npm run test -- --forbid-only --exit",
     "test:watch": "npm run test -- --watch",
```

Comment:
I assume now this command won't work. Maybe better `ts-node`?

Reply:
this works. Idea is source require conf file like list of plugins so the easiest way to run mocha tests is to run build and execute tests from dist

Reply:
can you explain how it work in `--watch` mode, if I change typescript file and tests are re-run?

Reply:
Watch mode in such setup won't work. I got your point.